src/main/java/frc/robot/RobotContainer.java

```diff
@@ -93,46 +94,58 @@ private void configureBindings() {
                   vision.camera.setPipelineIndex(4);
                   lights.setColorHSV(259, 100, 70);
                 }));
+    controller.button(54).whileTrue(new AutoLevelingCommand(drive));
+    controller.button(31).onTrue(new GrabManipulatorCommand(manipulator, GrabObject.CONE));
```

Comment:
Should be `whileTrue`

Reply:
`whileTrue` for the rest of the manipulator commands too, otherwise it never gets cancelled), though we should only be using the PID command in reality